src/test/java/com/amazon/janusgraph/triple/Triple.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2014-2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+package com.amazon.janusgraph.triple;
+
+import lombok.Getter;
+
+/**
+ *
+ * @author Addison Slabaugh
+ *
+ */
+public class Triple {
+
+    @Getter public static String[] line;
+    @Getter public static String leftObject;
+    @Getter public static String leftObjectProperty;
+    @Getter public static String relationship;
+    @Getter public static String rightObject;
+    @Getter public static String rightObjectProperty;
+
+    public Triple(String[] line) {
+        this.line = line;
+        processTriple(this.line);
```

Comment:
processTriple is just part of the logic of the constructor. Move the contents of processTriple to the constructor and then get rid of the processTriple method.